Palace/Utilities/UI/UINavigationBar+appearance.swift

```diff
@@ -19,4 +19,16 @@ extension UINavigationBar {
       compactScrollEdgeAppearance = appearance
     }
   }
+  
+  /// Updatign the appearance of the navbar, when presented, will not cause the changes to propogage.
+  /// Calling this function will trigger the view to redraw itself, forcing an appearance update
+  func forceUpdateAppearance(style: UIUserInterfaceStyle) {
+    _ = UIApplication.shared.windows.compactMap { window in
+      for view in window.subviews {
+        view.removeFromSuperview()
+        window.addSubview(view)
+        window.overrideUserInterfaceStyle = style
```

Comment:
this works! we just need to reset it when the user closes the book - or the style applies to the whole app

Reply:
Wow, missed that. I test in dark mode and kept switching back before navigating away from the title. I didn't realize the entire app changed. Will update!